test/azure/lro.ts

```diff
@@ -187,6 +187,21 @@ describe('typescript', function () {
       await testClient.lROs.post202NoRetry204({ product: product });
     });
 
+    // TODO coverage
+    it.skip('should work with LROPostDoubleHeadersFinalLocationGet', async () => {
+      await testClient.lROs.postDoubleHeadersFinalLocationGet({ product });
```

Comment:
Do these methods have return values? If so, can you check them? I understand that this currently just shows that the test doesn't throw an error, but it would be nice if they had some sort of expected result as well.

Reply:
I won't spend more time on the skipped tests for now-- this call on line 192 does not terminate--but whoever properly implements this test should check the return value if applicable